kubectl-cloudflow/cmd/deploy.go

```diff
@@ -387,7 +387,7 @@ func createOrUpdateCloudflowApplication(cloudflowApplicationClient *k8s.Cloudflo
 func validateDeployCmdArgs(cmd *cobra.Command, args []string) error {
 
 	if len(args) < 1 || args[0] == "" {
-		return fmt.Errorf("Please specify the full path to the Docker image containing the application. For example: 'docker-registry.mydomain.com/cloudflow/awesome-app:37-172e856'")
+		return fmt.Errorf("please specify the full path to the Docker image containing the application. For example: 'docker-registry.mydomain.com/cloudflow/awesome-app:37-172e856'")
```

Comment:
This is a new sentence, right?

Reply:
@agemooij Please take a look at https://github.com/golang/go/wiki/CodeReviewComments#error-strings .. golang encourages this convention :-)

Reply:
OK, I'd then like for us to check that the resulting log messages actually make sense, e.g. don't only change the initial capital but also check where the error message actually gets logged and then make sure the resulting sentence makes sense.